resources/assets/coffee/react/modding-profile/main.coffee

```diff
@@ -148,65 +149,66 @@ export class Main extends React.PureComponent
     profileOrder = @state.profileOrder
     isBlocked = _.find(currentUser.blocks, target_id: @state.user.id)
 
-    el DiscussionsContext.Provider,
-      value: @discussions()
-      el BeatmapsContext.Provider,
-        value: @beatmaps()
-        div
-          className: 'osu-layout__no-scroll' if isBlocked && !@state.forceShow
-          if isBlocked
-            div className: 'osu-page',
-              el NotificationBanner,
-                type: 'warning'
-                title: osu.trans('users.blocks.banner_text')
-                message:
-                  div className: 'grid-items grid-items--notification-banner-buttons',
-                    div null,
-                      el BlockButton, userId: @props.user.id
-                    div null,
-                      button
-                        type: 'button'
-                        className: 'textual-button'
-                        onClick: =>
-                          @setState forceShow: !@state.forceShow
-                        span {},
-                          i className: 'textual-button__icon fas fa-low-vision'
-                          " "
-                          if @state.forceShow
-                            osu.trans('users.blocks.hide_profile')
-                          else
-                            osu.trans('users.blocks.show_profile')
-
-          div className: "osu-layout osu-layout--full#{if isBlocked && !@state.forceShow then ' osu-layout--masked' else ''}",
-            el Header,
-              user: @state.user
-              stats: @state.user.statistics
-              userAchievements: @props.userAchievements
-
-            div
-              className: 'hidden-xs page-extra-tabs page-extra-tabs--profile-page js-switchable-mode-page--scrollspy-offset'
+    el ReviewEditorConfigContext.Provider, value: @props.reviewsConfig,
```

Comment:
Should probably at least be consistent with the other contexts below (i.e. value: ... on the following line, or changing them to match)